src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -128,7 +164,7 @@ public class RD2_OpportunityEvaluationService {
      * Active Schedule records.
      */
     @TestVisible
-    private Boolean isCandidate(npe03__Recurring_Donation__c rd) {
+    private Boolean isEligibleForEval(npe03__Recurring_Donation__c rd) {
```

Comment:
Based on [Handle Closed RD Open Opportunities PR](https://github.com/SalesforceFoundation/Cumulus/blob/e44ddc5294638632d6dcb540cd0b993c50eb3eea/src/classes/RD2_OpportunityEvaluationService.cls#L129), this method is removed, so due to renaming there might be merge conflict.

Reply:
Great catch. Thank you!